SUI/SUI/ContentView.swift

```diff
@@ -4,15 +4,182 @@
 
 import SwiftUI
 
+/// Экран настроек
 struct ContentView: View {
+    //    MARK: - Private Constatns
+    private enum Constants {
+        static let onOffBlueToothPickerValue = ["Выкл.", "Вкл."]
+        static let wifiPickerValues = ["my_wifi", "thisIsRouter", "122", "daire34"]
+        static let onOffPickerBluetoothIndexValue = 0
+        static let wifiIndexValue = 0
+        static let avatarImageCornerRadius = 40
+        static let avatarImageFrameWidth = 80
+        static let avatarImageFrameHeight = 80
+        static let avatarImageName = "ava"
+        static let userName = "Angelina Pozolotina"
+        static let userSettingsText = "Apple ID, iCloud, медиаконтент и покупки"
+        static let offerAppleText = "Предложения Apple ID"
+        static let freeOfferAppleText = "Бесплатные предложения от Apple"
+        static let newVersionText = "iOS 16.2 уже доступно"
+        static let memoryText = "Память iPhone заполнена"
+        static let aviaText = "Авиарежим"
+        static let wifiText = "Wi-Fi"
+        static let bluetoothTitle = "Bluetooth"
+        static let settingsTitle = "Настройки"
+        static let bluetoothImageName = "bluetooth"
+        static let userOfferFontSize = 11
+        static let chevronImageName = "chevron.right"
+        static let chevronImageFrameWidth = 7
+        static let chevronImageFrameHeight = 11
+        static let vpnTitle = "VPN"
+        static let vpnImageName = "vpn"
+        static let modemTitle = "Режим модема"
+        static let modemImageName = "modem"
+        static let cellTitle = "Сотовая связь"
+        static let cellImageName = "sot"
+        static let wifiImageName = "wifi"
+        static let aviaImageName = "avia"
+        static let twoString = "2"
+        static let oneString = "1"
+        static let imageCornerRadius = CGFloat(10)
+        static let circleWidth = CGFloat(25)
+        static let userSettingsLineLimitNumber = 1
+        static let userSettingsFontSize = CGFloat(11)
+        static let userNameFontSize = CGFloat(26)
+    }
+    
+    @State var wifiIndex = Constants.wifiIndexValue
+    @State var blueToothIndex = Constants.onOffPickerBluetoothIndexValue
+    @State var isOnAvia = false
+    @State var isOnVPN = false
+    var wifis = Constants.wifiPickerValues
```

Comment:
```suggestion

  private  var wifis = Constants.wifiPickerValues
```
Отступ между обычным вар и враперами и сделай их приватными тоже